.github/workflows/test.yml

```diff
@@ -46,6 +46,8 @@ jobs:
         run: yarn build
         env:
           NX_REJECT_UNKNOWN_LOCAL_CACHE: 0
+      - name: Check if there are changes
+        uses: NathanielHill/fail-if-changes@master
```

Comment:
Should be use `@latest` instead of `@master`? It would at least help in terms of clarification because our main branch has the same name and I was wondering if you were comparing anything to the `master` branch 😅

Reply:
`@latest` is not working but using a specifc version works 👍